index.html

```diff
@@ -5,46 +5,56 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>CC Globale Summit 2015</title>
-    <link rel="stylesheet" href="main.css" />
+    <title>IT.LIVE iraq 2019</title>
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/index.css" />
 </head>
 
 <body>
-    <header class="header flex">
-        <div class="header__nav hidden">
-            <a href="#"><img src="images/brands/airbnb.svg" alt="facebook" /></a>
-            <a href="#"><img src="images/brands/airbnb.svg" alt="twitter" /></a>
-            <div class="button-group">
+    <header class="header column">
+        <div class="header__nav flex bg-dark">
+            <a href="#"><img src="images/icons/facebook.svg" alt="facebook" /></a>
+            <a href="#"><img src="images/icons/twitter.svg" alt="twitter" /></a>
+            <div class="button-group flex">
+                <button type="button" class="btn btn-text">English</button>
                 <button type="button" class="btn btn-text">My Page</button>
                 <button type="button" class="btn btn-text">Logout</button>
             </div>
         </div>
-        <a class="hidden" href="index.html"><img src="" alt="cc global summit 2015 logo"></a>
-        <button type="button" class="btn btn-icon" data-toggle="collapse"><i class="icon-menu"></i></button>
-        <nav class="hidden">
-            <ul>
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
+        <div id="mobileNav" class="hidden">
+            <button id="btnClose" type="button" class="btn btn-icon close hidden" data-target="mobileMenuClose"><i
+                    class="icon-close"></i></button>
+            <ul id="mobileNavMenu" class="nav--mobile">
+                <li class="active"><a href="index.html">Home</a></li>
+                <li><a href="about.html">About</a></li>
+            </ul>
+        </div>
```

Comment:
- [ ] Please add this bottom to maintain the main [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
![Addbutton](https://user-images.githubusercontent.com/55596970/158559237-5a981e0a-8a81-4eee-8353-5c41ac39b4aa.png)